apispec/core.py

```diff
@@ -182,6 +182,13 @@ def add_path(self, path=None, operations=None, **kwargs):
     def definition(self, name, properties=None, enum=None, **kwargs):
         """Add a new definition to the spec.
 
+        If you are using the Marshmallow extension, you can pass fields'metadatas in the
```

Comment:
In a way yes; in another way, from the example in the README, spec.definition is used to reference a Marshmallow Schema; chances are people will stop there and probably not go deeper in the doc than spec.definition (just like I did).

If it were to be moved in ext.marshmallow's doc anyway, where would you put it ?